streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -160,15 +163,16 @@ private void addNewActiveTasks(final Map<TaskId, Set<TopicPartition>> newActiveT
     }
 
     private void addNewStandbyTasks(final Map<TaskId, Set<TopicPartition>> newStandbyTasks) {
-        log.trace("New standby tasks to be created: {}", newStandbyTasks);
+        log.debug("New standby tasks to be created: {}", newStandbyTasks);
```

Comment:
Not sure why this was trace as the same message is debug in the parallel active method

Reply:
Good catch!